code/script.js

```diff
@@ -30,18 +39,122 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// all the bot questions
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+    currentQuestion = 1 
```

Comment:
You don't really need this one here, it is already declared in the declaration on row 10